### ISSUE TITLE
Build warning/error when running project

### ISSUE THREAD

**Issue by larscom** [devmehq/react-qr-code#17]
## Bug report

- react version: 17.0+
- react-qr-code version: 1.0.5 

### Current behaviour
When running `react-scripts start` a build warning occurs:

```
WARNING in ./node_modules/@devmehq/react-qr-code/esm5/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/some/path/node_modules/@devmehq/src/index.tsx' file: Error: ENOENT: no such file or directory, open '/some/path/node_modules/@devmehq/src/index.tsx'
```
  

```tsx
import { ReactQrCode } from '@devmehq/react-qr-code';

<ReactQrCode size={200} value={'some_value'} />
```

### Expected behaviour

No build warning/error should occur when running `react-scripts start` created with `create-react-app`

```shell
npx create-react-app myproject --template typescript
```

**Comment by meabed** [comment]
Thank you Lars, that's related to the source maps, I'll publish them with the project and fix it.

**Comment by meabed** [comment]
@larscom I have fixed this in #1.0.6 https://www.npmjs.com/package/@devmehq/react-qr-code 
- please check it out and let me know. Thank you!
